app/src/main/res/values/strings.xml

```diff
@@ -1,91 +1,210 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-  <string name="nav_header_desc">Navigation header</string>
-  <string name="menu_options">Options</string>
-  <string name="menu_my_downloads">My Downloads</string>
-  <string name="menu_help">Help</string>
+  <!-- AdministratorControlActivity -->
+  <string name="administrator_controls_activity_label_title">Administrator Controls</string>
+  <!-- AdminAuthActivity -->
+  <string name="admin_auth_activity_toolbar_title">Administrator Controls</string>
+  <string name="admin_auth_activity_add_profiles_title">Authorise to add profiles</string>
+  <string name="admin_auth_activity_access_controls_title">Authorise to access Administrator Controls</string>
+  <string name="admin_auth_activity_heading">Administrator Authorization Required</string>
+  <string name="admin_auth_activity_admin_auth_sub_text">Enter the Administrator PIN in order to create a new account.</string>
+  <string name="admin_auth_activity_admin_auth_admin_controls_sub_text">Enter the Administrator PIN in order to access Administrator Controls.</string>
+  <string name="admin_auth_activity_admin_auth_pin_input_hint_text">Administrator\'s PIN</string>
+  <string name="admin_auth_activity_admin_auth_incorrect_pin_error_message">Incorrect Administrator PIN. Please try again.</string>
+  <string name="admin_auth_activity_admin_auth_null_pin_error_message">Please enter Administrator PIN.</string>
+  <string name="admin_auth_activity_submit_button_text">Submit</string>
+  <string name="admin_auth_activity_close_button_text">Close</string>
+  <!-- AdminPinActivity -->
+  <string name="save">Save</string>
+  <string name="admin_pin_activity_admin_pin_requirement_description">Before we add profiles, we need to protect your account by creating a PIN. This gives you the ability to authorize downloads and manage profiles on the device.</string>
+  <string name="admin_pin_activity_admin_pin_warning_text">Do <b>not</b> use a PIN you have set for personal accounts like banking or social security.</string>
+  <string name="admin_pin_activity_admin_pin_input_hint_text">New 5-Digit PIN</string>
+  <string name="admin_pin_activity_admin_pin_input_confirm_pin_text">Confirm 5-Digit PIN</string>
+  <string name="admin_pin_activity_admin_pin_length_error_text">Your PIN should be 5 digits long.</string>
+  <string name="admin_pin_activity_wrong_confirm_admin_pin_error_text">Please make sure that both PINs match.</string>
+  <string name="admin_pin_activity_label_text">Authorise to add profiles</string>
+  <!-- AdministratorControlsFragment -->
+  <string name="administrator_controls_fragment_test_activity_label">Administrator Controls Fragment Test Activity</string>
```

Comment:
Could you move this & other test-only strings to https://github.com/oppia/oppia-android/blob/develop/app/src/main/res/values/untranslated_strings.xml? They don't need to be translated.